### DISCUSSION TITLE
Can't Start Netbox - New Install/New User/

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#9382] [Q&A] [answered]
I have just installed NetBox (yesterday).  I had it up and running and was populating the database to make sure everything was working okay.  I rebooted the node and now can't get NetBox to start up.

gunicorn[20206]: PermissionError: [Errno 13] Permission denied: '/opt/netbox-3.2.3/venv/lib/python3.9/site-packages/pyasn1_modules-0.2.8.dist-info'
systemd[1]: netbox.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: netbox.service: Failed with result 'exit-code'.

I've tried CHMOD 755 and CHOWN netbox:netbox with no success
drwxr-xr-x  2 netbox netbox     122 May 17 00:34 pyasn1-0.4.8.dist-info

pyasn1-0.4.8.dist-info]# ls -l
total 32
-rwxr-xr-x 1 netbox netbox    4 May 17 00:34 INSTALLER
-rwxr-xr-x 1 netbox netbox 1334 May 17 00:34 LICENSE.rst
-rwxr-xr-x 1 netbox netbox 1521 May 17 00:34 METADATA
-rwxr-xr-x 1 netbox netbox 5360 May 17 00:34 RECORD
-rwxr-xr-x 1 netbox netbox    7 May 17 00:34 top_level.txt
-rwxr-xr-x 1 netbox netbox  110 May 17 00:34 WHEEL
-rwxr-xr-x 1 netbox netbox    1 May 17 00:34 zip-safe

and I've tried CHMOD 600 and CHOWN root:root like the other folders
drw-------  2 root root     122 May 17 00:34 pyasn1-0.4.8.dist-info

pyasn1-0.4.8.dist-info
total 32
-rw------- 1 root root    4 May 17 00:34 INSTALLER
-rw------- 1 root root 1334 May 17 00:34 LICENSE.rst
-rw------- 1 root root 1521 May 17 00:34 METADATA
-rw------- 1 root root 5360 May 17 00:34 RECORD
-rw------- 1 root root    7 May 17 00:34 top_level.txt
-rw------- 1 root root  110 May 17 00:34 WHEEL
-rw------- 1 root root    1 May 17 00:34 zip-safe

Does anyone have any suggestions or guidance for me please?

**Comment by candlerb** [comment, marked as answer]
> and I've tried CHMOD 600 and CHOWN root:root like the other folders

That will make things worse, not better.

If you are running in the standard way, you'll find netbox is running as the "netbox" user. That means that this user must be able to read all the python library files.

The files should be mode 644, and the directories should be mode 755, to make them accessible to all users.

```
find /opt/netbox -type f -print0 | xargs -0 chmod 644
find /opt/netbox -type d -print0 | xargs -0 chmod 755
```

It's not really important whether they're owned by root:root or netbox:netbox.  (I leave them mostly owned as root:root for slightly better security, however the media directory has to be owned by netbox:netbox to allow it to upload image attachments)

**Reply by ghost** [reply]
Thanks Brian.  That fix led me to two more directories that required the same fix.  I'm up and running now.  Much  appreciation.
